toolkits/postgres/arcade_postgres/tools/database_engine.py

```diff
@@ -88,3 +76,29 @@ async def cleanup(cls) -> None:
     def clear_cache(cls) -> None:
         """Clear the engine cache without disposing engines. Use with caution."""
         cls._engines.clear()
+
+    @classmethod
+    def sanitize_query(cls, query: str) -> str:
```

Comment:
I understand this isn't going to be for 'production', but maybe at least additionally check for write keywords like DELETE, INSERT & error if they are in the str? Idk up to you